mobify/platform-scaffold
# [IM Extensibility] Checkout custom results and reducers

This PR adds results, reducers and selectors for custom content in the various branches of the checkout state. 

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-75

## Changes
- Added results, reducers & selectors for checkout shipping branch
- Added results, reducers & selectors for checkout shipping address branch
- Added results, reducers & selectors for checkout billing branch
- Added results, reducers & selectors for checkout billing address branch
- Added results, reducers & selectors for checkout payment branch
- Added results, reducers & selectors for checkout payment address branch
- Added results, reducers & selectors for checkout locations branch

## How to test-drive this PR
- Run `npm run test`
- All the tests should pass
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- the site should still work as expected
